docs/configuration.md

```diff
@@ -201,4 +202,14 @@ var provider = new playkit.providers.ott.Provider(config);
 > ##### Type: `boolean`
 >
 >
-> ##### Description: Instructs the player to ignore the server configuration.
\ No newline at end of file
+> ##### Description: Instructs the player to ignore the server configuration.
+
+> ##
+>
+> ### config.loadThumbnailWithKs
+>
+> ##### Type: `boolean`
+>
+> ##### Default: `false`
+>
+> ##### Description: Defines whether to add KS to sprite and poster requests or not.>
```

Comment:
keep it generic for all api call

Reply:
"Defines whether to add KS to thumbnail api request or not" is good?

Reply:
and add this is for ovp only